zookeeper-server/src/main/resources/lib/jetty-client-9.4.35.v20201120.LICENSE.txt

```diff
@@ -1,8 +1,7 @@
 This program and the accompanying materials are made available under the
-terms of the Eclipse Public License 1.0 which is available at
-https://www.eclipse.org/org/documents/epl-1.0/EPL-1.0.txt
-or the Apache Software License 2.0 which is available at
-https://www.apache.org/licenses/LICENSE-2.0
+terms of the Eclipse Public License 2.0 which is available at
```

Comment:
@phunt: This patch is for `master`, which still pulls `jetty-client`; I have noted that it should not be included in `branch-3.5`.

@eolivelli: Yes, will do so.  And `branch-3.6`, too, as it does not cherry-pick clean.

Reply:
(@phunt: In case you were suggesting to remove `jetty-client` from the POM in `master`, that would break [ZOOKEEPER-3948: Introduce a deterministic runtime behavior injection framework for ZooKeeperServer testing](https://github.com/apache/zookeeper/commit/4432f5b44359).)